topbeat/docs/fields.asciidoc

```diff
@@ -328,6 +328,27 @@ type: float
 The percentage of used swap memory.
 
 
+==== swap.actual_used
```

Comment:
Good catch. It looks like somehow they are not deleted from etc/fields.yml. I checked gosigar code and the Swap structure contains only the information: Total, Used, Free. Could you please delete the rest of fields in this PR? Thanks